README.md

```diff
@@ -157,5 +157,5 @@ Tabs are indexed starting from 0.
 The following API methods are provided:
 
 * `init()`: Set attributes/classes, bind events. Called automatically on construction. Does nothing if already been called.
-* `selectTab(idx)`: Select tab `idx`. Does nothing if tab `idx` does not exist or is already selected.
+* `selectTab(idx, disableFocus)`: Select tab `idx`. Does nothing if tab `idx` does not exist or is already selected. The optional 'disableFocus' argument can be set to true to stop the aria-selected tab receving focus if required.
```

Comment:
Can you mention that it prevents unexpected scrolling from happening?

Also, prefer the active tense to the passive tense, it's easier to understand (especially for the native French and Spanish speakers in the Origami team :fr: :es:):
`do - this - to perform - that - ` (e.g. "Set `w` as `x` to stop y and prevent z")